olive/extra_dependencies.json

```diff
@@ -12,7 +12,7 @@
         "onnxruntime"
     ],
     "gpu": [
-        "onnxruntime-gpu"
+        "onnxruntime-gpu>=1.17.0"
```

Comment:
I don't think we should pin the version here. the version requirements are example/pass specific. 

Instead, you can just add the required packages for the new pass directly to `pass_to_extra` instead of `pass_to_extra_names`.

Reply:
yeah, better not to pin the version from production code.